### PR TITLE
Allow configuration of opcache maximum files for the cache

### PR DESCRIPTION
**Pull request by cuppett**
Introduce environment variable for PHP 7.2, 7.3 and 7.4. OPCACHE_MAX_FILES, controls opcache.max_accelerated_files setting which needs customized for some PHP applications or to be able to tune the cache effectively. The default is set to the old, hardcoded value we had in here.

See also: [NextCloud Server Tuning - Enable PHP OPcache](https://docs.nextcloud.com/server/19/admin_manual/installation/server_tuning.html#enable-php-opcache)

### CODE REVIEW

**Comment by rhscl-automation-poll** [comment]
Can one of the admins verify this patch?

**Comment by rhscl-automation-poll** [comment]
Can one of the admins verify this patch?

**Comment by rhscl-automation-poll** [comment]
Can one of the admins verify this patch?

**Comment by rhscl-automation-poll** [comment]
Can one of the admins verify this patch?

**Comment by cuppett** [comment]
@hhorak do I need to do 7.2 here as well?

File: 7.3/s2i/bin/run [hunk 1]
```diff
@@ -26,6 +26,7 @@ export SHORT_OPEN_TAG=${SHORT_OPEN_TAG:-OFF}
 export OPCACHE_MEMORY_CONSUMPTION=${OPCACHE_MEMORY_CONSUMPTION:-128}
 
 export OPCACHE_REVALIDATE_FREQ=${OPCACHE_REVALIDATE_FREQ:-2}
+export OPCACHE_MAX_FILES=${OPCACHE_REVALIDATE_FREQ:-4000}
```
Comment (hhorak) [inline 7.3/s2i/bin/run:29]: A copy&paste issue here.

**Comment by hhorak** [comment]
> @hhorak do I need to do 7.2 here as well?

PHP 7.2 is maintained for one more month, so it would be better. Thanks.

File: 7.3/s2i/bin/run (hunk 1 above)
Comment (cuppett) [reply 7.3/s2i/bin/run]: Fixed, thanks!

**Comment by cuppett** [comment]
> > @hhorak do I need to do 7.2 here as well?
> 
> PHP 7.2 is maintained for one more month, so it would be better. Thanks.

Done. 7.2 as well. Thanks!

**Comment by hhorak** [comment]
Thanks, LGTM now. Merging.

**Pull request merged**